### PR TITLE
chore: changing event to emittery and updating all packages

### PR DESCRIPTION
**Pull request by johnsre**

### CODE REVIEW

**Comment by ktrz-sre-is** [comment]
Hey @johnsre 
Doing code formatting on the whole repo makes it super hard to review the PR. Could you please revert that part?

**Comment by ktrz-sre-is** [comment]
> Hey @johnsre Doing code formatting on the whole repo makes it super hard to review the PR. Could you please revert that part?

Actually, I went ahead and created a separate PR with setting up and running prettier. I went for the same configurations as in our other projects so I had to modify eslint config slightly to not collide

PR [here](https://github.com/StanzaSystems/stanza-js/pull/212)

File: packages/node/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@connectrpc/connect-node": "~1.1.2",
     "@bufbuild/protobuf": "~1.4.2",
     "@grpc/grpc-js": "~1.9.10",
-    "emittery": "^0.13.1",
+    "emittery": "^1.0.1",
```
Comment (ktrz-sre-is) [inline packages/node/package.json:28]: this package needs to stay on 0.13.1 because of some trouble with module resolution

Comment (johnsre) [reply packages/node/package.json]: Okay, I just reverted it!

File: package.json [hunk 2]
```diff
@@ -3,116 +3,115 @@
...
-    "@nx/react": "17.1.1",
-    "@nx/rollup": "17.1.1",
-    "@nx/vite": "17.1.1",
-    "@nx/web": "17.1.1",
-    "@nx/workspace": "17.1.1",
-    "@testing-library/react": "14.0.0",
-    "@types/cors": "^2.8.13",
-    "@types/express": "~4.17.17",
-    "@types/jest": "^29.5.2",
+    "@babel/preset-react": "^7.23.3",
```
[the remark this answers is not in the document]
Comment (johnsre) [reply package.json]: I reverted this one too.

[the remark this answers is not in the document]
Comment (ktrz-sre-is) [reply package.json]: there are still some package upgrades that seem unrelated 🤔

File: packages/core/src/eventEmitter.test.ts [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
-      eventEmitter.dispatchChange('My change');
+    expect(async () => {
+      await eventEmitter.emit('change', 'My change');
     }).not.toThrow();
   });
 
-  it('should listen to a dispatched change event', () => {
-    const eventEmitter = new StanzaChangeTarget<string>();
+  it('should listen to a dispatched change event', async () => {
+    const eventEmitter = new Emittery();
```
[the remark this answers is not in the document]
Comment (johnsre) [reply packages/core/src/eventEmitter.test.ts]: Fixed

**Pull request closed without merging**